File: src/backend/Models/Translation.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Threading.Tasks;

namespace Sleet.Models
{
    /// <summary>
    /// Translation是单条翻译，需要成组使用
    /// </summary>
    public class Translation
    {
        public Guid Id { get; set; }
        public string Key { get ; set ; }
        public bool IsOriginalLanguage { get; set; }
        public Language Language { get; set; }
        public string Content { get; set; }
        public List<Suggestion> Suggestions { get; set; }
    }
}

File: src/frontend/src/services/openapi/models/Translation.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Language } from './Language';
import type { Suggestion } from './Suggestion';

export type Translation = {
    id?: string;
    key?: string | null;
    isOriginalLanguage?: boolean;
    language?: Language;
    content?: string | null;
    suggestions?: Array<Suggestion> | null;
}